### PR TITLE
Adds alt-clicking to open containers

### PR DESCRIPTION
**Pull request by Shadowtail117**
You can now alt-click a container to open it, following the same rules as click-dragging a container to open it. I.e., no being incapacitated and within one tile. Because QoL is good.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/storage.dm [hunk 1]
```diff
@@ -93,6 +93,15 @@
 				usr.equip_to_slot_if_possible(src, slot_l_hand)
 		src.add_fingerprint(usr)
 
+/obj/item/storage/AltClick(var/mob/usr)
+	if(!canremove)
+		return ..()
+	if ((ishuman(usr) || isrobot(usr) || issmall(usr)) && !usr.incapacitated() && Adjacent(usr))
```
Comment (Geevies) [inline code/game/objects/items/weapons/storage/storage.dm:99]: i think you should be able to replace all of this with if(!use_check_and_message(usr)), would look a lot nicer

File: code/game/objects/items/weapons/storage/storage.dm [hunk 2]
```diff
@@ -93,6 +93,15 @@
...
+		src.add_fingerprint(usr)
```
Comment (Geevies) [inline code/game/objects/items/weapons/storage/storage.dm:100]:
```suggestion
		add_fingerprint(usr)
```

File: code/game/objects/items/weapons/storage/storage.dm [hunk 3]
```diff
@@ -93,6 +93,15 @@
...
+		src.open(usr)
```
Comment (Geevies) [inline code/game/objects/items/weapons/storage/storage.dm:101]:
```suggestion
		open(usr)
```

**Review by Geevies — APPROVED** [review]

**Comment by Shadowtail117** [comment]
!review

**Review by JohnWildkins — CHANGES_REQUESTED** [review]
Pretty cool QoL. Unfortunately, this does not account for `obj/item/storage/secure` subtypes, so right now you can take stuff out by alt-clicking on secure briefcases and safes.

Probably should just override AltClick() on the `/secure` subtype and check if the storage is locked before opening.

**Comment by Shadowtail117** [comment]
Good catch, patched.

**Comment by Shadowtail117** [comment]
!review

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**